src/components/slack/list/slacklist.component.ts

```diff
@@ -500,6 +511,19 @@ export class SlackListComponent implements OnInit, OnDestroy {
         }
     }
 
+    onMouseEnterReaction(reaction: DisplaySlackReactionInfo) {
+        reaction.showReactionUsers = true;
```

Comment:
If mouse cursor is away from the reaction before callback function is invoked, this flag will be true. So cannot move it I think.